tools/cldr-code/src/main/java/org/unicode/cldr/test/CLDRTest.java

```diff
@@ -922,10 +925,13 @@ public void TestMinimalLocalization() throws IOException {
                                     new String[] {
                                         "sun", "mon", "tue", "wed", "thu", "fri", "sat"
                                     }));
-            for (int i = -7; i < 0; ++i) {
-                checkForItems(item, (i < -4 ? months : days), i, missing, failureCount, null);
```

Comment:
This usage of the magic number "-7" to mean "NameType.TZ_EXEMPLAR and getDateKey", and so forth, involved some tricky arithmetic. This PR reduces the trickiness, at the expense of making the parameters explicit instead of looping.

Reply:
Add this as a comment to the code -- otherwise it hard for us in the future to follow why this is.

A practice I liked, is to move magic numbers to the top of the file as constant integers so when you call on them you call on English-readable labels.

Reply:
Fortunately my change here removes the magic numbers, so I don't think there's any need for a comment

`for (int i = -7; i < 0; ++i) {` is the old code being replaced

Reply:
agreed, this is a review on the old code. no comment required.